manifests/charts/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -81,6 +81,10 @@ spec:
       serviceAccountName: istiod{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}
 {{- if .Values.global.priorityClassName }}
       priorityClassName: "{{ .Values.global.priorityClassName }}"
+{{- end }}
```

Comment:
Why would we add this to `istiod` only? Why (or why not) add this to every chart? It's general enough.